### ISSUE TITLE
Consider to remove anystyle generated id field

### ISSUE THREAD

**Issue by mdoering** [Sp2000/colplus-backend#103]
Anystyle generates an id value for all parsed CSL references to be used for bibliography links. It is not based on existing ids but the author and year. Probably better to remove such generated ids.

Explicitly given ids via ReferenceFactory params such as DOI should be part of the Reference instance, but not part of the CSL.

**Comment by ayco-at-naturalis** [comment]
Yes, I suggested either removing the ID, or put the dataset-provided IDs back onto the CSL. I will nullify the CSL IDs
